academy/2-cosmos-concepts/15-bridges.md

```diff
@@ -14,7 +14,7 @@ tags:
 
 Make sure to read the following sections as a preparation:
 
-* [The Interchain Ecosystem](/academy/1-what-is-cosmos/2-cosmos-ecosystem.md)
+* [The interchain ecosystem](/academy/1-what-is-cosmos/2-cosmos-ecosystem.md)
```

Comment:
Here I would leave it in upper case because it is a section title and should follow the same capitalization.